pootle/apps/pootle_config/managers.py

```diff
@@ -93,7 +93,7 @@ def list_config(self, key=None, model=None):
                 conf = conf.filter(key__in=key)
         elif key is not None:
             conf = conf.filter(key=key)
-        for item in conf:
+        for item in conf.order_by("key", "pk"):
```

Comment:
because there may be multiple k/v pairs with same k, this ensures it gets the last - which is at least predictable